public/userview.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <!-- Referenced https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsDataURL -->
+    <script>
+        function showImg() {
+            const preview = document.querySelector('img');
+            const file = document.querySelector('input[type=file]').files[0];
+            const reader = new FileReader();
+            reader.addEventListener("load", function () {
+                // convert image file to base64 string
+                preview.src = reader.result;
+            }, false);
+
+            if (file) {
+                reader.readAsDataURL(file);
+            }
+        }
+    </script>
+
+    <!-- Referenced https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch -->
+    <script>
+        function upload() {
+            document.getElementById("form1").addEventListener("click", function(event){
+                event.preventDefault()
+            });
+
+            const formData = new FormData();
+            const fileField = document.querySelector('input[type="file"]');
+            formData.append('image', fileField.files[0]);
+
+            fetch('http://localhost:8765', {
+                method: 'PUT',
+                body: formData
+            })
+            .then(response => response.text())
+            .then(classification => document.getElementById("p1").textContent = classification);
+        }
+    </script>
+
+    <form id="form1" action="javascript:;" onsubmit="upload(this)"></form>
+        <label for="img">Select image:</label>
+        <input type="file" accept="image/*" onchange="showImg()">
```

Comment:
give this `id="img"` so that the label above's `for` works properly

Reply:
done